contracts/Tokenomics.sol

```diff
@@ -1,7 +1,6 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.25;
```

Comment:
Not sure we should change Tokenomics at all. Feels like we could get away with keeping the existing implementation ...

Reply:
Dev incentives revision force us to change tokenomics anyway.

Reply:
I see. However, ETH is still being used as part of the donation and distributed to all components/agents. Is it distributed according to the same proportion as the boosts?

Reply:
The ETH distribution is going to be left as is. Meaning that proportions will be different.